### PR TITLE
Allow Invoke commands to be run on Linux/Mac

### LINKED ISSUE #105: Fix failing tests on Windows 11 (ARM emulated through Parallels on Mac M1)
Failing tests:
```
FAILED tests/robocorp_windows_tests/test_robocorp_windows_chrome.py::test_multiple_interactions
FAILED tests/robocorp_windows_tests/test_robocorp_windows_explorer.py::test_copy_with_explorer
```

All of them ending in an `ElementNotFound` error due to system's UI different layouts. If not simple/stable, then mark the tests as `@xfail`.

### LINKED ISSUE #264: Make `robocorp-windows` develop-able on Mac/Linux as well
As there's no need for Windows OS to update a README, bumping the version and making a release for e.g.

Similarly to what we have in RPA Framework:
- `RPA.Windows`.[`__init__.py`](https://github.com/robocorp/rpaframework/blob/master/packages/windows/src/RPA/Windows/__init__.py#L12-L19)
- `RPA.core.windows`.[`context.py`](https://github.com/robocorp/rpaframework/blob/master/packages/core/src/RPA/core/windows/context.py#L7-L9)

### PR DESCRIPTION
**Pull request by cmin764**
Closes #264 #105

Minimum intrusion overrides in the absence of the Windows OS during importing execution so most of the Invoke commands can be run.

Generated docs as well and it didn't deviate from the originals in content.

### CODE REVIEW

File: windows/docs/api/.pages
Comment (cmin764) [inline windows/docs/api/.pages:1]: Have no idea about this mode, but locally I get the docs right:
```
% git diff master -- windows/docs/api/robocorp.windows.md
diff --git a/windows/docs/api/robocorp.windows.md b/windows/docs/api/robocorp.windows.md
old mode 100755
new mode 100644
```

**Comment by cmin764** [comment]
And type-check [fails](https://github.com/robocorp/robocorp/actions/runs/8615396795/job/23610850294?pr=339):
Screenshot 2024-04-09 at 4 21 23 PM

File: windows/docs/api/.pages
Comment (OvidiuCode) [reply windows/docs/api/.pages:1]: Seems like it is related to file permissions mode on each system, I think we can ignore that.

**Review by OvidiuCode — APPROVED** [review]

**Review by cmin764 — COMMENTED** [review]
LGTM!

**Pull request merged**